mk/doc-gen.py

```diff
@@ -34,4 +37,5 @@
         t['src'] = '//golang.org' + t['src']
 
     # Write updated HTML to stdout
-    print(soup.prettify().encode('utf-8'))
+    print(soup.prettify())
```

Comment:
Why'd you drop the utf8 conversion?
Does this still work on both py2 and p3?

Reply:
`print(string.encode('utf-8'))` prints a bytes literal in Python 3. It works in Python 3 now (I think it won't in Python 2 though).